src/stores/security-two-factor-authentication-use.md

```diff
@@ -53,13 +40,74 @@ With the U2F device configured, the user attaches the device to their system whe
 
 1. When complete, click <span class="btn">Confirm</span>.
 
+## Duo Security
+
+The Duo Mobile app is available through [Google Play][3] or [iOS App Store][4]. To use, it must be activated and linked to an account. For details, your security staff should obtain a Duo account and complete setup.
+
+### Step 1: Configure Duo Security
+
+1. Enter your account credentials and sign in to the Magento _Admin_.
+
+1. When the Duo Setup page appears. Click **Start setup** and do the following:
+
+    ![Example storefront - Duo setup]({% link images/images/storefront-2fa-duo-user1.png %}){: .zoom}
+    _Duo Setup_
+
+   - Select your device.
+
+        ![Duo authentication - select device ]({% link images/images/storefront-2fa-duo-user2.png %}){: .zoom}
+        _Device Type_
+
+   - When prompted, enter your **phone number**, and click **Continue**.
+
+        This example requests your phone number, because we are using a mobile device.
```

Comment:
should mobile device be smart phone?

Reply:
It's OK as is. Duo Mobile uses "mobile device"